### ISSUE TITLE
Allow defining database server hostname and port

### ISSUE THREAD

**Issue by ToeiRei** [shlinkio/shlink#77]
bin/install never asks about the MySQL host to use. Assume it assumes localhost.

**Comment by acelaya** [comment]
That's a good point.
I'll fix it

**Comment by ToeiRei** [comment]
Anything to hack around that in the meanwhile?

**Comment by acelaya** [comment]
Sure.
If I remember correctly, you should be able to set both host and/or port in the config/autoload/entity_manager.global.php

```php
<?php
return [

    'entity_manager' => [
        'orm' => [
            'proxies_dir' => 'data/proxies',
        ],
        'connection' => [
            'driver' => 'pdo_mysql',
            'user' => env('DB_USER'),
            'password' => env('DB_PASSWORD'),
            'dbname' => env('DB_NAME', 'shlink'),
            'charset' => 'utf8',
            'host' => '1.2.3.4', // <- Add this if you need to change the host
            'port' => 33306, // <- Add this if you need to change the port
            'driverOptions' => [
                PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES utf8',
            ],
        ],
    ],

];
```

Just remember to delete the `data/cache/app_config.php` file so that it gets regenerated, otherwise your changes won't take effect.

**Comment by acelaya** [comment]
Already implemented https://github.com/acelaya/shlink/tree/feature/1.3.1
Will be available in shlink 1.3.1
